pages/common/virt-install.md

```diff
@@ -0,0 +1,8 @@
+# virt-install
+
+> libvirt utility to create new virtual machine and run installation.
+> Home page: <https://virt-manager.org/>.
+
+- Create a virtual machine with 1G RAM and 12G storage and run Debian installation:
+
+`virt-install --memory {{1024}} --disk path={{~/image.qcow2}},size={{12}} --cdrom {{/path/to/file/debian.iso}}`
```

Comment:
Very minor thing really - just for consistency with other pages :-)
```suggestion
`virt-install --memory {{1024}} --disk path={{path/to/image.qcow2}},size={{12}} --cdrom {{path/to/file/debian.iso}}`
```

Reply:
I was afraid of long-long line, but I am fully agreed if you find it OK. And an absolute path came from [qemu-img](../blob/master/pages/common/qemu-img.md) page.